src/components/PerrsonLink/PersonLink.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonLink: React.FC<Props> = React.memo(
+  ({ person }) => (
+    <>
+      <Link
+        to={`../${person.slug}`}
```

Comment:
now when you click on person name there're showing the 404 page. it's because of the wrong link here
```suggestion
        to={`/people/${person.slug}`}
```

Reply:
https://mate-parttime.slack.com/archives/C041JA5EJKV/p1677053593954679  already discused. It doesn't work